### PR TITLE
Un-vendor ai_horde

### PR DESCRIPTION
**Pull request by valadaptive**
The `@zeldafan0225/ai_horde` package was previously vendored because one of its dependencies was trying to build itself every time it installed, which failed on Termux because the build tool didn't work on Android.

I submitted [a PR to that dependency which has now been merged](https://github.com/thunder04/supermap/pull/1), so we can once again un-vendor `ai_horde` and manage it through npm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
         "@agnai/sentencepiece-js": "^1.1.1",
         "@agnai/web-tokenizers": "^0.1.3",
         "@dqbd/tiktoken": "^1.0.13",
+        "@zeldafan0225/ai_horde": "^4.0.1",
+        "ai-horde": "^0.1.0",
```
Comment (Cohee1207) [inline package.json]: Is this a mistake install?

Comment (valadaptive) [reply package.json]: Yup; my bad

**Comment by Cohee1207** [comment]
It's good to merge, but we've lost one custom addition to ai_horde (euler_a sampler for image generation).
But I think it should be pushed upstream. 

UPD: PR created to ai_horde https://github.com/ZeldaFan0225/ai_horde/pull/12

**Comment by Cohee1207** [comment]
Verified working on Android

**Pull request merged**